tests/lib/snaps/test-snapd-base-bare/meta/snap.yaml

```diff
@@ -0,0 +1,4 @@
+name: test-snapd-base-bare
```

Comment:
Can't comment on the empty files/dirs above, so adding here:

Wouldn't it be better to create them dynamically inside the test instead of having these in the tree?

Reply:
Yes, thank you! I added a snapcraft.yaml that creates them now instead of manually doing it here.